DANS-KNAW/easy-deposit-api
# EASY-2457 send deposit confirmation mail

Fixes EASY-2457 send deposit confirmation mail

#### When applied it will
* 
* 
* 

#### Where should the reviewer @DANS-KNAW/easy start?

#### How should this be manually tested?

* Build, deploy (also a recent version of `easy-agreement-creator`, `easy-agreement-generator`) 
* Login on https://deasy.dans.knaw.nl/deposit with a user that has your email configured and submit a dataset. 

Note that the email may or may not be marked as spam by the knaw mail server. Attached pdf should have the same content as the agreement on the administration tab.

Pushing the commit button caused the easy-deposit-api to log something like shown below. This shows copying 141 payload files and creating the metadata took about half a second, generating the agreement took almost 4 seconds.

    [2019-12-16 09:37:32,448] INFO  request PUT .../metadata ...
    [2019-12-16 09:37:32,488] INFO  response PUT .../metadata ...
    [2019-12-16 09:37:32,497] INFO  request PUT .../state ...
    [2019-12-16 09:37:32,969] INFO  calling easy-deposit-agreement-generator ...
    ...                             creating manifest
    [2019-12-16 09:37:36,369] INFO  email placeholder values: ...
    [2019-12-16 09:37:36,691] INFO  moving ...
    [2019-12-16 09:37:36,875] INFO  sent email <...4@deasy>
    [2019-12-16 09:37:36,879] INFO  response PUT .../state ...
    [2019-12-16 09:37:36,924] INFO  request GET http://localhost:20190/deposit ...
    [2019-12-16 09:37:37,001] INFO  response GET http://localhost:20190/deposit ...


#### Related pull requests on github
* [ ] dtap?
